.github/workflows/ci-pull-request.yml

```diff
@@ -22,7 +22,6 @@ jobs:
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip
           pip install wheel
```

Comment:
Should this have an `--upgrade` flag as well? Same with most of the other yml files

Reply:
Included `pip install --upgrade wheel` .